### ISSUE TITLE
Some correntions for the wiki page "Build ePMC under Debian based Linux Distributions"

### ISSUE THREAD

**Issue by JonisK** [ISCAS-PMC/ePMC#6]
Here are some errata and my fixes to make the build succeed when following the wiki page [Build ePMC under Debian based Linux Distributions](https://github.com/ISCAS-PMC/ePMC/wiki/Build-ePMC-under-Debian-based-Linux-Distributions)

Would be great if you could implement those.

The git URL is wrong. Use
```
git clone https://github.com/ISCAS-PMC/ePMC.git`
```

The `mv` command expects a path that is not present yet and keeps the unnecessary `jdk-14.0.1` folder. Replace it with these commands:
```
sudo mkdir /usr/lib/jvm
sudo mkdir /usr/lib/jvm/oracle-jdk
sudo mv jdk-14.0.1/* /usr/lib/jvm/oracle-jdk
```
(The current jdk version is 14.0.2 so feel free to update that, too)

Very minor thing: Before running `build.sh`, change to the ePMC directory, e.g.:
```
cd ~/ePMC
```

**Comment by andreat** [comment]
Hi,
thanks for pointing out these mistakes.

> The `mv` command expects a path that is not present yet and keeps the unnecessary `jdk-14.0.1` folder. Replace it with these commands:
> 
> ```
> sudo mkdir /usr/lib/jvm
> sudo mkdir /usr/lib/jvm/oracle-jdk
> sudo mv jdk-14.0.1/* /usr/lib/jvm/oracle-jdk
> ```

I changed your commands to 
```
sudo mkdir -p /usr/lib/jvm/
sudo mv jdk-14.0.2 /usr/lib/jvm/oracle-jdk
```

The `-p` option in the first command prevents `mkdir` to complain in case the directory already exists, as well as to create the parent directories (but this is not the case here).
The second command merges your two commands: it moves `jdk-14.0.2` inside `/usr/lib/jvm/`after renaming it as `oracle-jdk`, if it does not exist, otherwise `jdk-14.0.2` is moved as it is inside `/usr/lib/jvm/oracle-jdk`. I added a note about the case where `/usr/lib/jvm/oracle-jdk` already exists.
